### ISSUE TITLE
SwiftLint config error on carthage build

### ISSUE THREAD

**Issue by toshi0383** [ReactKit/SwiftState#45]
On `carthage build` or `carthage update` , this library currently emits swiftlint config errors.

```
$ carthage build --platform tvos SwiftState
*** xcodebuild output can be found in /var/folders/tr/3lftss053tqdh731cgt0w1p00000gn/T/carthage-xcodebuild.eJzwNs.log
*** Building scheme "SwiftState" in SwiftState.xcodeproj
2016-03-05 01:41:43.498 xcodebuild[19404:463121] ### Failed to load Addressbook class CNContactNameFormatter
2016-03-05 01:41:48.926 xcodebuild[19466:463272] ### Failed to load Addressbook class CNContactNameFormatter
configuration error: 'variable_name_min_length' is not a valid rule identifier
configuration error: 'variable_name_min_length' is not a valid rule identifier
```

IMO, a library should not be running SwiftLint in its Build Phase.

**Comment by inamiy** [comment]
Thanks for issue! I fixed it in #46.

**Comment by toshi0383** [comment]
🚀
